server/lib/job-queue/job-queue.ts

```diff
@@ -124,6 +124,12 @@ class JobQueue {
 
     for (const handlerName of (Object.keys(handlers) as JobType[])) {
       const queue = new Bull(handlerName, queueOptions)
+
+      if (produceOnly) {
+        queue.pause(true)
```

Comment:
Don't we need to resume it somewhere as well?

Reply:
No because this instance of the job queue for the current process is just to produce new tasks in the job queue. 

(we use the `local` boolean https://github.com/OptimalBits/bull/blob/develop/lib/queue.js#L805)

(I may have misunderstood your comment)